lib/util/support.js

```diff
@@ -126,6 +126,12 @@ function mobileWebKitBrowser(browser) {
   }) || null;
 }
 
+// Browser detection to check for whether or not the browser is Chrome iOS.
+// eslint-disable-next-line no-warning-comments
+// TODO(joyce): Move browser detection to a different util
+const isAndroid = /android/.test(navigator.userAgent);
```

Comment:
This is duplicated in the isSupported function. We should create `isAndroid` function and reuse that.